commit 0c9e4fc2ad6d88d54f299240fcc5a2ce7d695d96
Author: Jia Tan <jiat0218@gmail.com>
Date:   Sat Jan 21 21:32:03 2023 +0800

    liblzma: Fix documentation for LZMA_MEMLIMIT_ERROR.
    
    LZMA_MEMLIMIT_ERROR was missing the "<" character needed to put
    documentation after a member.

diff --git a/src/liblzma/api/lzma/base.h b/src/liblzma/api/lzma/base.h
index caf488fe..9e513450 100644
--- a/src/liblzma/api/lzma/base.h
+++ b/src/liblzma/api/lzma/base.h
@@ -139,7 +139,7 @@ typedef enum {
 		 */
 
 	LZMA_MEMLIMIT_ERROR     = 6,
-		/**
+		/**<
 		 * \brief       Memory usage limit was reached
 		 *
 		 * Decoder would need more memory than allowed by the